x-pack/plugins/security/server/audit/audit_service.ts

```diff
@@ -87,9 +87,11 @@ export class AuditService {
     getSpaceId,
   }: AuditServiceSetupParams): AuditServiceSetup {
     if (config.enabled && !config.appender) {
-      this.licenseFeaturesSubscription = license.features$.subscribe(({ allowAuditLogging }) => {
-        this.allowAuditLogging = allowAuditLogging;
-      });
+      this.licenseFeaturesSubscription = license.features$.subscribe(
+        ({ allowLegacyAuditLogging }) => {
+          this.allowAuditLogging = allowLegacyAuditLogging;
```

Comment:
nit: Renaming this variable to distinguish between legacy and non-legacy would improve readability a bit:

```suggestion
          this.allowLegacyAuditLogging = allowLegacyAuditLogging;
```